vscode-typescript/01-passing-tests/setup-container

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+echo "Running challenge setup script on host container"
+
+# Install Node.js
+# https://github.com/nodesource/distributions/blob/master/README.md
+curl -fsSL https://deb.nodesource.com/setup_14.x | bash -
+apt-get install -y nodejs
+
+# Clone the example repository
+git clone https://github.com/borodean/typescript-example-app.git /app
```

Comment:
I have forked the repo under the Instruqt org: https://github.com/instruqt/typescript-example-app

Reply:
@borodean Can you update this URL to the fork on the instruqt org?

Reply:
Updated: https://github.com/instruqt/templates/pull/3/commits/642f75d1c098aebd3a8f35e9dc6ac9a88ec92b4c.